tests/integrationtest/lib.go

```diff
@@ -922,25 +922,30 @@ func NormalControllerServer(w http.ResponseWriter, r *http.Request, args ...stri
 	addrPrefix := "10.250.250"
 	object := strings.Split(strings.Trim(url, "/"), "/")
 
-	if strings.Contains(url, "macro") && r.Method == "POST" {
+	if r.Method == "POST" && !strings.Contains(url, "login") {
 		data, _ := ioutil.ReadAll(r.Body)
 		json.Unmarshal(data, &resp)
-		rData, rModelName := resp["data"].(map[string]interface{}), strings.ToLower(resp["model_name"].(string))
-		rName := rData["name"].(string)
+		_, rModelName := resp, ""
```

Comment:
Minor: we can simplify this line as rModelName := "" ... as resp is not considered